docs/index.html

```diff
@@ -19,12 +19,17 @@
       loadNavbar: false,
       maxDepth: 3,
       subMaxLevel: 3,
-      basePath: 'https://raw.githubusercontent.com/rcos/rcos-handbook/master/docs',
+      //basePath: 'https://raw.githubusercontent.com/rcos/rcos-handbook/master/docs',
```

Comment:
Oops! This needs to be uncommented for the site to work in production - but in development you need it commented out - it's pretty annoying :/

Reply:
Good catch!